code/__HELPERS/game.dm

```diff
@@ -1,5 +1,5 @@
 //This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:31
-#define DEATHTIME_XENO_REQUIREMENT 3000
+#define DEATHTIME_XENO_REQUIREMENT 5 MINUTES
```

Comment:
what even uses this define

Reply:
```		
//Recently dead observers cannot be drafted.
		var/deathtime = world.time - O.timeofdeath
		if(deathtime < DEATHTIME_XENO_REQUIREMENT)
			continue
```

Reply:
how is this different from away timer?

Reply:
This is about candidate selection. the away is about eligible xenos for takeover.

This means that you need to be a ghost for 5 minutes before you can take a xeno whos afk for 5 mins

Reply:
Can you delete this define and use `GLOB.respawntime` instead?

Reply:
Sure